File: examples/notes/list/node/listAllNotes.js
import * as notes from "m3o/notes";

// List all the notes
async function ListAllNotes() {
  let notesService = new notes.NotesService(process.env.MICRO_API_TOKEN);
  let rsp = await notesService.list({});
  console.log(rsp);
}

await ListAllNotes();

File: examples/go/notes/list/listAllNotes.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/notes"
)

// List all the notes
func ListAllNotes() {
	notesService := notes.NewNotesService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := notesService.List(&notes.ListRequest{})
	fmt.Println(rsp, err)
}
